### PR TITLE
2 packages from bcpierce00/unison at 2.53.4

### PR DESCRIPTION
**Pull request by jhjourdan**
This pull-request concerns:
- `unison.2.53.4`: File-synchronization tool for Unix and Windows
- `unison-gui.2.53.4`: Pseudo-package for Unison GUI

---
* Homepage: https://www.cis.upenn.edu/~bcpierce/unison/
* Source repo: git://github.com/bcpierce00/unison.git
* Bug tracker: https://github.com/bcpierce00/unison/issues

---
:camel: Pull-request generated by opam-publish v2.3.0

### CODE REVIEW

**Comment by mseri** [comment]
`unison-gui` is failing consistently with
```
# ocamlopt -g -I lwt -I ubase -I system -I +unix -I +str -I system/generic -I lwt/generic -I +lablgtk3 -I +cairo2 -c /home/opam/.opam/4.08/.opam-switch/build/unison.2.53.4/src/uigtk3.ml
# File "/home/opam/.opam/4.08/.opam-switch/build/unison.2.53.4/src/uigtk3.ml", line 75, characters 26-48:
# 75 | let fontMonospace = lazy (Pango.Font.from_string "monospace")
#                                ^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound module Pango
```
Ping @garrigue (maybe you know why)

**Comment by jhjourdan** [comment]
Ocamlfind was a missing dependency of `unison-gui`. This should be fixed now.

**Comment by jhjourdan** [comment]
Is there an Opam variable telling whether the native OCaml compiler is installed?

Or is there a way to tell in the opam file that we require a native build of OCaml?

File: packages/unison/unison.2.53.4/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+build: [make "-j" jobs]
+install: [make "PREFIX=%{prefix}%" "install"]
+depends: [
+  "ocaml" {>= "4.08"}
+]
+depopts: [
+  "lablgtk3" {>= "3.1.0"}
+  "ocamlfind"
+]
+synopsis: "File-synchronization tool for Unix and Windows"
```
[the remark this answers is not in the document]
Comment (jhjourdan) [reply packages/unison/unison.2.53.4/opam:21]: Thanks, but I instead used the `ocaml:native` variable.

**Comment by jhjourdan** [comment]
The Makefile of unison requires XCode to build the native Mac GUI, and apparently the CI does not have XCode installed. I don't have an Apple machine to test whether the Opam package works under MacOS.

I have two solutions here: either I deactivate support for MacOS in the Opam package, or I trust that it's just going to work if XCode is installed, and I let you merge even if CI does not pass. If people have issues with the Opam package under MacOS, they'll have to file a bug report and we'll fix the Opam package afterwards.

What do you think?

**Comment by mseri** [comment]
I think it's ok to keep it as it is

**Comment by mseri** [comment]
Thanks!

**Pull request merged**